common/filter-tags/outputs.tf

```diff
@@ -0,0 +1,85 @@
+output "filter_custom" {
+  description = "The full filtering pattern to use in detectors"
+  value = "${var.filter_use_defaults == true ? var.filter_defaults :
+    "${var.filter_custom_includes == "" ? join(
+      " and ",
+      formatlist(
+        "(not filter('%s'))",
+        split(
+          ";",
+          replace(
+            replace(
+              var.filter_custom_excludes,
+              ":",
+              "', '"
+            ),
+            " ",
+            ""
+          )
+        )
+      )
+      )
+      :
+      "${var.filter_custom_excludes == "" ? join(
+        " and ",
+        formatlist(
+          "(not filter('%s'))",
+          split(
+            ";",
+            replace(
+              replace(
+                var.filter_custom_includes,
+                ":",
+                "', '"
+              ),
+              " ",
+              ""
+            )
+          )
+        )
+        )
+        :
+        join(
+          " and ",
+          list(
+            join(
+              " and ",
+              formatlist(
+                "filter('%s')",
+                split(
+                  ";",
+                  replace(
+                    replace(
+                      var.filter_custom_includes,
+                      ":",
+                      "', '"
+                    ),
+                    " ",
+                    ""
+                  )
+                )
+              )
+            ),
+            join(
+              " and ",
+              formatlist(
+                "(not filter('%s'))",
+                split(
+                  ";",
+                  replace(
+                    replace(
+                      var.filter_custom_excludes,
+                      ":",
+                      "', '"
+                    ),
+                    " ",
+                    ""
+                  )
+                )
+              )
+            )
+          )
+      )}"
+    }"
+  }"
```

Comment:
```suggestion
  value = (
    var.filter_use_defaults == true ?
    var.filter_defaults
    :
    var.filter_custom_includes == "" ?
    join(
      " and ",
      formatlist(
        "(not filter('%s'))",
        split(
          ";",
          replace(
            replace(
              var.filter_custom_excludes,
              ":",
              "', '"
            ),
            " ",
            ""
          )
        )
      )
    )
    :
    var.filter_custom_excludes == "" ?
    join(
      " and ",
      formatlist(
        "(not filter('%s'))",
        split(
          ";",
          replace(
            replace(
              var.filter_custom_includes,
              ":",
              "', '"
            ),
            " ",
            ""
          )
        )
      )
    )
    :
    join(
      " and ",
      list(
        join(
          " and ",
          formatlist(
            "filter('%s')",
            split(
              ";",
              replace(
                replace(
                  var.filter_custom_includes,
                  ":",
                  "', '"
                ),
                " ",
                ""
              )
            )
          )
        ),
        join(
          " and ",
          formatlist(
            "(not filter('%s'))",
            split(
              ";",
              replace(
                replace(
                  var.filter_custom_excludes,
                  ":",
                  "', '"
                ),
                " ",
                ""
              )
            )
          )
        )
      )
    )
  )
```

Reply:
oh my bad, I did a wrong test and missing to replace the negative part .. thanks!